components/palette.css

```diff
@@ -51,16 +51,11 @@
 .textDanger   { color: var(--textDangerColor) }
 .textMuted    { color: var(--textMutedColor) }
 
+/* TODO - needs to be refactored since this file shouldn't just be a place to stuff classes */
```

Comment:
Yea I like that as well, however I did notice that some of these classes seem to be used in only one place. Those are the ones I would want to move.